src/i18n/es.js

```diff
@@ -240,6 +240,7 @@ export default {
     url_site: `${urlSiteFromSignup}`,
   },
   trafficSources: {
+    error: `¡Ouch! Ocurrio un error inesperado, vuelva a intentar nuevamente`,
```

Comment:
Pehaps this general error could be added in validation messages for common usage, what do you think?

Reply:
we talk in slack, for now use this until have final contents